scripts/octoeverywhere.sh

```diff
@@ -273,9 +273,51 @@ function remove_octoeverywhere()
 #===================================================#
 
 function update_octoeverywhere() {
-  # Since our update might require new python packages or system packages, ask the user to use the
-  # moonraker update manager to do it, since that takes care of it.
-  print_error "Please use the Moonraker Update Manager from the Mainsail or Fluidd UI to update OctoEverywhere.\n Contact our support team if you need help! support@octoeverywhere.com"
+  do_action_service "stop" "octoeverywhere"
```

Comment:
Will this handle the octoeveryhere-* other possible service names?

Reply:
Yes, calling that function handles all services beginning with "octoeverywhere".